routes.js

```diff
@@ -43,28 +28,15 @@ function fastifySwagger (fastify, opts, next) {
     url: '/documentation',
     method: 'GET',
     schema: { hide: true },
-    handler: (request, reply) => reply.redirect(request.raw.url + '/')
+    handler: (request, reply) => reply.redirect('./documentation/')
   })
 
-  fastify.route({
-    url: '/documentation/:file',
-    method: 'GET',
-    schema: { hide: true },
-    handler: sendStaticFiles
+  // server swagger-ui with the help of fastify-static
```

Comment:
typo, it should be "serve swagger-ui with the help of fastify-static".

Reply:
fixed